Makefile

```diff
@@ -210,7 +213,7 @@ ifeq (,$(shell which opm 2>/dev/null))
 	set -e ;\
 	mkdir -p $(dir $(OPM)) ;\
 	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
-	curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/v1.15.1/$${OS}-$${ARCH}-opm ;\
+	curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/v1.20.0/$${OS}-$${ARCH}-opm ;\
```

Comment:
I hope you verified the compatibility with this new version.

Reply:
I didn't find any compatibility matrix for opm and operator-sdk or olm. However I did use this new version to create registry image for operator testing, didn't see any issues while creation or running the registry image. 

Also this change has nothing to do with the conversion webhook or crd upgrade. I made it for my local test environment as older version didn't support mac(arm). If needed, we can always revert this change.